hw/ip_templates/rv_plic/fpv/vip/rv_plic_assert_fpv.sv.tpl

```diff
@@ -104,4 +105,7 @@ module ${module_instance_name}_assert_fpv #(parameter int NumSrc = 1,
   // but smaller than the threshold
   `ASSERT(IdChangeWithIrq_A, !$stable(irq_id_o[tgt_sel]) && irq_id_o[tgt_sel] != 0 |->
           irq_o[tgt_sel] || ((irq_id_o[tgt_sel]) == $past(i_high_prio) && !$past(irq)))
+
+  // When fatal alert happens then only reset can clear it.
+  `ASSERT(FatalAlertNeverLow_A, ##1 !$fell(fatal_alert_i))
```

Comment:
Less nitty: I think you probably want something like "never drops" instead of "never low". Indeed, we expect it to be low most of the time :-)